presets/4.3/tune/fpvian_basher_freestyle.txt

```diff
@@ -0,0 +1,181 @@
+#$ TITLE: Basher Freestyle 5" by FPVian
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: basher, freestyle, 5 Inch, 5 In, 5", bando, fpvian
+#$ AUTHOR: FPVian
+#$ DESCRIPTION: Want to be able to take your quad to the BANDO and still have it fly well?
+#$ DESCRIPTION: Want a preset that you can load on to all your 5" Freestyle quads?
+#$ DESCRIPTION: Don't want to give up the feeling of flying a well-tuned machine?
+#$ DESCRIPTION:
+#$ DESCRIPTION: Well look no further! Guaranteed to contain a filter preset that works for you.
+#$ DESCRIPTION: Don't like it? Go fix your bashed-up quad!
+#$ DESCRIPTION: 
+#$ DESCRIPTION: Optimized for AUW: ~700g (including HD cam), but anything should work fine.
+#$ DESCRIPTION: The "Filtering for CLEAN builds" option is recommended for best overall performance.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Leaving all filter options unchecked will leave the filters unchanged.
+#$ DESCRIPTION: You MUST enable bidirectional DSHOT300 or DSHOT600 to use the filtering options.
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ DISCUSSION: 
+
+# Set to Defaults
+
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+
+# Defaults end
+
+
+#PRESETS START
+
+set tpa_mode = PD
+set anti_gravity_gain = 4200
+set iterm_relax_cutoff = 10
+
+set simplified_i_gain = 135
+set simplified_pi_gain = 110
+set simplified_dmax_gain = 0
+set simplified_feedforward_gain = 140
+set simplified_pitch_pi_gain = 105
+simplified_tuning apply
+
+
+#$ OPTION BEGIN (CHECKED): Set Dynamic Idle to 30
+set dyn_idle_min_rpm = 30
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Increase Feedforward Jitter Reduction to 13
```

Comment:
is there a specific reason for jitter factor set here?
it is more of an rc_link setting and depends on what radio link person flies as far as I understand. @ctzsnooze ?

Reply:
Specified for HD Recording in Jitter Reduction Option